tests/connectors/kubernetes_test.py

```diff
@@ -495,6 +496,25 @@ def test_strategy_object_canary(self) -> None:
             "  max: 4\n"
         )
 
+    def test_strategy_object_none(self) -> None:
+        config = DeploymentConfiguration(
+            name="testing",
+            containers=[],
+            replicas=servo.Replicas(min=1, max=4),
+            strategy=NoOptimizationStrategyConfiguration(
+                type=OptimizationStrategy.none
+            ),
+        )
+        assert config.yaml(exclude_unset=True) == (
+            "name: testing\n"
+            "containers: []\n"
+            "strategy:\n"
+            "  type: none\n"
+            "replicas:\n"
+            "  min: 1\n"
+            "  max: 4\n"
+        )
```

Comment:
What's this block supposed to test? If you're looking to ensure the object gets created, isn't simply not raising exceptions enough? This string matching stuff is kinda brittle.